### ISSUE TITLE
%EnvironmentVariables% Not Interpreted in External Path for Private Key

### ISSUE THREAD

**Issue by rjt** [dlech/KeeAgent#252]
I have per user environment windows variables that are ignored by keepass / keepassAgent 0.10.1 resulting in  "Could Not Find FIle" errors.   
#148

**Comment by dlech** [comment]
Where are you using this environment variable?

**Comment by rjt** [comment]
In a keepass record called MyDropboxRsaKey, the KeeAgent tab has an "External File" radio and text box.  In the "External File" text box, i have %DROPBOX%\MyDropboxRsaKey.   

KeePass Installed via Chocolatey.   OS=Win10 1803 17134.407  
Below is the error:

KeeAgent: Error while loading key from entry 'MyPassWords\MyDropboxRsaKey'
Could not find file
%DROPBOX%\MyDropboxRsaKey

**Comment by rjt** [comment]
Forgive me for thinking Windows 10 17134.407 would behave the same as one of the previous versions, probably 17133 in which user based environment variables seemed to be more consistent after closing out of all cmd.exe and powershell.exe shells and keepass.   WinFlag+R recognized %DropBox%, so why would not an application launched from the start menu.  I had restarted KeePass.exe and actually made sure it was closed numerous times via process monitoring.  Executed RefreshEnv numerous times.  Sorry to bother you guys.   My Keys are loading now:)
